lib/rmt/mirror.rb

```diff
@@ -191,7 +191,7 @@ def replace_directory(source_dir, destination_dir)
 
     FileUtils.remove_entry(old_directory) if Dir.exist?(old_directory)
     FileUtils.mv(destination_dir, old_directory) if Dir.exist?(destination_dir)
-    FileUtils.mv(source_dir, destination_dir)
+    FileUtils.mv(source_dir, destination_dir, force: '-f')
```

Comment:
Nitpicking: looking at the documentation, wouldn't "force: true" also work? (in fact, the example on the documentation is passing a boolean value)

Reply:
Yes, makes sense, changed. I think I copied that from the proposal in Bugzilla.